Gemfile

```diff
@@ -94,7 +94,6 @@ group :development do
   gem "graphql-docs"
   gem "i18n-tasks"
   gem "listen" # Using with `rails s` since Rails 5
-  gem "memory_profiler"
```

Comment:
Why the memory_profiler was removed?

Reply:
@shimbaco It is already included on this line:
https://github.com/annict/annict/blob/b5d2741f4ebe969994ce158cf64982a3c26210a5/Gemfile#L45

Bundler will return an error if you don't remove one.

Reply:
oh, I didn't notice that!